packages/login/src/vaadin-lit-login-overlay.js

```diff
@@ -55,6 +56,10 @@ class LoginOverlay extends LoginOverlayMixin(ElementMixin(ThemableMixin(PolylitM
           @disabled-changed="${this._onDisabledChanged}"
         ></vaadin-login-form>
       </vaadin-login-overlay-wrapper>
+
+      <div hidden>
+        <slot id="fields" name="custom-fields"></slot>
```

Comment:
Does it need `id="fields"`?

```suggestion
        <slot name="custom-fields"></slot>
```

Reply:
Removed.

Reply:
Removed.